From 2ad02c84f62060fccabd3c337fb715a2d5a0b358 Mon Sep 17 00:00:00 2001
From: Shunqing Chen <csq@rock-chips.com>
Date: Wed, 11 Aug 2021 09:24:20 +0800
Subject: [PATCH 059/232] media: rk628-for-all: modify edid

1. recommended Resolution set to 1080P60.
2. add 1080P50 support.
3. modify vendor/ product information.

Signed-off-by: Shunqing Chen <csq@rock-chips.com>
Change-Id: Ic355377e53e07ec36be84726e1acb86a8e3a5506
Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
---
 drivers/media/i2c/rk628/rk628_bt1120_v4l2.c |  8 ++++----
 drivers/media/i2c/rk628/rk628_csi.c         | 22 ++++++++++-----------
 drivers/media/i2c/rk628/rk628_csi_v4l2.c    | 22 ++++++++++-----------
 3 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/drivers/media/i2c/rk628/rk628_bt1120_v4l2.c b/drivers/media/i2c/rk628/rk628_bt1120_v4l2.c
index 70439dab8038c7..9765585f2a9bf1 100644
--- a/drivers/media/i2c/rk628/rk628_bt1120_v4l2.c
+++ b/drivers/media/i2c/rk628/rk628_bt1120_v4l2.c
@@ -167,15 +167,15 @@ static u8 edid_init_data[] = {
 	0x10, 0x10, 0x3E, 0x96, 0x00, 0x13, 0x8E, 0x21,
 	0x00, 0x00, 0x1E, 0xD8, 0x09, 0x80, 0xA0, 0x20,
 	0xE0, 0x2D, 0x10, 0x10, 0x60, 0xA2, 0x00, 0xC4,
-	0x8E, 0x21, 0x00, 0x00, 0x18, 0x8C, 0x0A, 0xD0,
-	0x90, 0x20, 0x40, 0x31, 0x20, 0x0C, 0x40, 0x55,
-	0x00, 0x48, 0x39, 0x00, 0x00, 0x00, 0x18, 0x01,
+	0x8E, 0x21, 0x00, 0x00, 0x18, 0x02, 0x3A, 0x80,
+	0xD0, 0x72, 0x38, 0x2D, 0x40, 0x10, 0x2C, 0x45,
+	0x80, 0x20, 0xC2, 0x31, 0x00, 0x00, 0x1E, 0x01,
 	0x1D, 0x80, 0x18, 0x71, 0x38, 0x2D, 0x40, 0x58,
 	0x2C, 0x45, 0x00, 0xC0, 0x6C, 0x00, 0x00, 0x00,
 	0x18, 0x01, 0x1D, 0x80, 0x18, 0x71, 0x1C, 0x16,
 	0x20, 0x58, 0x2C, 0x25, 0x00, 0xC0, 0x6C, 0x00,
 	0x00, 0x00, 0x18, 0x00, 0x00, 0x00, 0x00, 0x00,
-	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x39,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x45,
 };
 
 static const struct rk628_bt1120_mode supported_modes[] = {
diff --git a/drivers/media/i2c/rk628/rk628_csi.c b/drivers/media/i2c/rk628/rk628_csi.c
index 8ad91cd966a289..485adcf1d52792 100644
--- a/drivers/media/i2c/rk628/rk628_csi.c
+++ b/drivers/media/i2c/rk628/rk628_csi.c
@@ -113,8 +113,8 @@ struct rk628_csi_mode {
 
 static u8 edid_init_data[] = {
 	0x00, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x00,
-	0x52, 0x62, 0x01, 0x88, 0x00, 0x88, 0x88, 0x88,
-	0x1C, 0x15, 0x01, 0x03, 0x80, 0x00, 0x00, 0x78,
+	0x49, 0x73, 0x8D, 0x62, 0x00, 0x88, 0x88, 0x88,
+	0x08, 0x1E, 0x01, 0x03, 0x80, 0x00, 0x00, 0x78,
 	0x0A, 0x0D, 0xC9, 0xA0, 0x57, 0x47, 0x98, 0x27,
 	0x12, 0x48, 0x4C, 0x00, 0x00, 0x00, 0x01, 0x01,
 	0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01,
@@ -127,24 +127,24 @@ static u8 edid_init_data[] = {
 	0x37, 0x34, 0x39, 0x2D, 0x66, 0x48, 0x44, 0x37,
 	0x32, 0x30, 0x0A, 0x20, 0x00, 0x00, 0x00, 0xFD,
 	0x00, 0x14, 0x78, 0x01, 0xFF, 0x1D, 0x00, 0x0A,
-	0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x01, 0x7B,
+	0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x01, 0x18,
 
 	0x02, 0x03, 0x1A, 0x71, 0x47, 0x5F, 0x90, 0x22,
 	0x04, 0x11, 0x02, 0x01, 0x23, 0x09, 0x07, 0x01,
 	0x83, 0x01, 0x00, 0x00, 0x65, 0x03, 0x0C, 0x00,
-	0x10, 0x00, 0x8C, 0x0A, 0xD0, 0x8A, 0x20, 0xE0,
-	0x2D, 0x10, 0x10, 0x3E, 0x96, 0x00, 0x13, 0x8E,
-	0x21, 0x00, 0x00, 0x1E, 0xD8, 0x09, 0x80, 0xA0,
-	0x20, 0xE0, 0x2D, 0x10, 0x10, 0x60, 0xA2, 0x00,
-	0xC4, 0x8E, 0x21, 0x00, 0x00, 0x18, 0x8C, 0x0A,
-	0xD0, 0x90, 0x20, 0x40, 0x31, 0x20, 0x0C, 0x40,
-	0x55, 0x00, 0x48, 0x39, 0x00, 0x00, 0x00, 0x18,
+	0x10, 0x00, 0x02, 0x3A, 0x80, 0x18, 0x71, 0x38,
+	0x2D, 0x40, 0x58, 0x2C, 0x45, 0x00, 0x20, 0xC2,
+	0x31, 0x00, 0x00, 0x1E, 0x01, 0x1D, 0x00, 0x72,
+	0x51, 0xD0, 0x1E, 0x20, 0x6E, 0x28, 0x55, 0x00,
+	0x20, 0xC2, 0x31, 0x00, 0x00, 0x1E, 0x02, 0x3A,
+	0x80, 0xD0, 0x72, 0x38, 0x2D, 0x40, 0x10, 0x2C,
+	0x45, 0x80, 0x20, 0xC2, 0x31, 0x00, 0x00, 0x1E,
 	0x01, 0x1D, 0x80, 0x18, 0x71, 0x38, 0x2D, 0x40,
 	0x58, 0x2C, 0x45, 0x00, 0xC0, 0x6C, 0x00, 0x00,
 	0x00, 0x18, 0x01, 0x1D, 0x80, 0x18, 0x71, 0x1C,
 	0x16, 0x20, 0x58, 0x2C, 0x25, 0x00, 0xC0, 0x6C,
 	0x00, 0x00, 0x00, 0x18, 0x00, 0x00, 0x00, 0x00,
-	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xD8,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC1,
 };
 
 static const struct rk628_csi_mode supported_modes[] = {
diff --git a/drivers/media/i2c/rk628/rk628_csi_v4l2.c b/drivers/media/i2c/rk628/rk628_csi_v4l2.c
index b5456b446344cc..c358a004773336 100644
--- a/drivers/media/i2c/rk628/rk628_csi_v4l2.c
+++ b/drivers/media/i2c/rk628/rk628_csi_v4l2.c
@@ -169,8 +169,8 @@ static const struct v4l2_dv_timings_cap rk628_csi_timings_cap = {
 
 static u8 edid_init_data[] = {
 	0x00, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x00,
-	0x52, 0x62, 0x01, 0x88, 0x00, 0x88, 0x88, 0x88,
-	0x1C, 0x15, 0x01, 0x03, 0x80, 0x00, 0x00, 0x78,
+	0x49, 0x73, 0x8D, 0x62, 0x00, 0x88, 0x88, 0x88,
+	0x08, 0x1E, 0x01, 0x03, 0x80, 0x00, 0x00, 0x78,
 	0x0A, 0x0D, 0xC9, 0xA0, 0x57, 0x47, 0x98, 0x27,
 	0x12, 0x48, 0x4C, 0x00, 0x00, 0x00, 0x01, 0x01,
 	0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01,
@@ -183,24 +183,24 @@ static u8 edid_init_data[] = {
 	0x37, 0x34, 0x39, 0x2D, 0x66, 0x48, 0x44, 0x37,
 	0x32, 0x30, 0x0A, 0x20, 0x00, 0x00, 0x00, 0xFD,
 	0x00, 0x14, 0x78, 0x01, 0xFF, 0x1D, 0x00, 0x0A,
-	0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x01, 0x7B,
+	0x20, 0x20, 0x20, 0x20, 0x20, 0x20, 0x01, 0x18,
 
 	0x02, 0x03, 0x1A, 0x71, 0x47, 0x5F, 0x90, 0x22,
 	0x04, 0x11, 0x02, 0x01, 0x23, 0x09, 0x07, 0x01,
 	0x83, 0x01, 0x00, 0x00, 0x65, 0x03, 0x0C, 0x00,
-	0x10, 0x00, 0x8C, 0x0A, 0xD0, 0x8A, 0x20, 0xE0,
-	0x2D, 0x10, 0x10, 0x3E, 0x96, 0x00, 0x13, 0x8E,
-	0x21, 0x00, 0x00, 0x1E, 0xD8, 0x09, 0x80, 0xA0,
-	0x20, 0xE0, 0x2D, 0x10, 0x10, 0x60, 0xA2, 0x00,
-	0xC4, 0x8E, 0x21, 0x00, 0x00, 0x18, 0x8C, 0x0A,
-	0xD0, 0x90, 0x20, 0x40, 0x31, 0x20, 0x0C, 0x40,
-	0x55, 0x00, 0x48, 0x39, 0x00, 0x00, 0x00, 0x18,
+	0x10, 0x00, 0x02, 0x3A, 0x80, 0x18, 0x71, 0x38,
+	0x2D, 0x40, 0x58, 0x2C, 0x45, 0x00, 0x20, 0xC2,
+	0x31, 0x00, 0x00, 0x1E, 0x01, 0x1D, 0x00, 0x72,
+	0x51, 0xD0, 0x1E, 0x20, 0x6E, 0x28, 0x55, 0x00,
+	0x20, 0xC2, 0x31, 0x00, 0x00, 0x1E, 0x02, 0x3A,
+	0x80, 0xD0, 0x72, 0x38, 0x2D, 0x40, 0x10, 0x2C,
+	0x45, 0x80, 0x20, 0xC2, 0x31, 0x00, 0x00, 0x1E,
 	0x01, 0x1D, 0x80, 0x18, 0x71, 0x38, 0x2D, 0x40,
 	0x58, 0x2C, 0x45, 0x00, 0xC0, 0x6C, 0x00, 0x00,
 	0x00, 0x18, 0x01, 0x1D, 0x80, 0x18, 0x71, 0x1C,
 	0x16, 0x20, 0x58, 0x2C, 0x25, 0x00, 0xC0, 0x6C,
 	0x00, 0x00, 0x00, 0x18, 0x00, 0x00, 0x00, 0x00,
-	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xD8,
+	0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC1,
 };
 
 static const struct rk628_csi_mode supported_modes[] = {
-- 
2.17.1

